android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -869,6 +949,17 @@ private WritableMap buildParticipant(Participant participant) {
         return participantMap;
     }
 
+
+    private WritableMap buildParticipantDataEvent(Participant participant) {
+        WritableMap participantMap = buildParticipant(participant);
+        WritableMap participantMap2 = buildParticipant(participant);
```

Comment:
Shouldn't these be the fields for the track?

Reply:
Perhaps you can help here? Not sure how to pass this kind of track and output the relevant information important to this type of track.

I'm sorry this event information just isn't important to me right now.